app/lib/actions/katello/repository_set/scan_cdn.rb

```diff
@@ -17,9 +17,9 @@ class ScanCdn < Actions::AbstractAsyncTask
         end
 
         def plan(product, content_id)
-          prod_content = product.product_content_by_id(content_id).content
+          prod_content = product.product_content_by_id(content_id).content # is this necessary?
```

Comment:
what specifically are you asking about?  this is needed to be passed into the action as its used in the content method of this action.

Reply:
Ah! That content_id *is* the cp_content_id and on plan_self we are taking it, looking up the Product Content using that ID, and pulling the same ID off of the Content object. The comment was more for me to follow up on. I'm removing that lookup.